tools/SourceKit/include/SourceKit/Core/LangSupport.h

```diff
@@ -1189,16 +1189,11 @@ class LangSupport {
                                 ArrayRef<const char *> Args,
                                 std::function<void(const RequestResult<ArrayRef<StringRef>> &)> Receiver) = 0;
 
-  virtual void syntacticRename(llvm::MemoryBuffer *InputBuf,
-                               ArrayRef<RenameLocations> RenameLocations,
-                               ArrayRef<const char*> Args,
-                               CategorizedEditsReceiver Receiver) = 0;
-
   virtual void findRenameRanges(llvm::MemoryBuffer *InputBuf,
                                 ArrayRef<RenameLocations> RenameLocations,
                                 ArrayRef<const char *> Args,
                                 CategorizedRenameRangesReceiver Receiver) = 0;
-  virtual void
+   virtual void
```

Comment:
Accidental? Does clang-format not remove this?

Reply:
It does. I just didn’t push after running `git-clang-format` 🙈